cds/factory.py

```diff
@@ -54,7 +54,7 @@
 """
 
 create_celery = create_app_factory(
-    'cds',
+    'cds_celery',
```

Comment:
Well, just to clarify which contexts you are in. The API renaming is mandatory, as they are both `Flask apps` and its creates great confusion in debugging. In the case of Celery, it's not that confusing, but still it is more clear that way.